tests/rules/test_git_branch_exists.py

```diff
@@ -17,18 +17,25 @@ def new_command(branch_name):
         'git branch -D {0} && git checkout -b {0}', 'git checkout {0}']]
 
 
-@pytest.mark.parametrize('script, branch_name', [
-    ('git branch foo', 'foo'), ('git checkout bar', 'bar')])
+@pytest.mark.parametrize('script, src_branch_name, branch_name', [
+    ('git branch foo', 'foo', 'foo'),
+    ('git checkout bar', 'bar', 'bar'),
+    ('git checkout -b "let\s-push-this"', '"let\s-push-this"', '"let\s-push-this"')])
```

Comment:
I think there are some `'` missing here and in lines bellow.

Reply:
Thanks, yes, that'll tech me to find/replace!